src/makeCalculator.js

```diff
@@ -1,10 +1,34 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate: (callback, number) => {
+      callback(number);
+
+      return calculator;
+    },
+    add: (number) => {
+      calculator.result += number;
+    },
+    subtract: (number) => {
+      calculator.result -= number;
+    },
+    multiply: (number) => {
+      calculator.result *= number;
+    },
+    divide: (number) => {
+      calculator.result /= number;
```

Comment:
The `divide` method should include a check to prevent division by zero, which would result in an error or an undefined result. Consider adding a condition to handle this case.